Oddities.WinHelp/Oddities.WinHelp.csproj

```diff
@@ -12,11 +12,19 @@
         <Description>WinHelp (.hlp) file format reader for .NET.</Description>
     </PropertyGroup>
 
+    <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+      <LangVersion>preview</LangVersion>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+      <LangVersion>preview</LangVersion>
+    </PropertyGroup>
```

Comment:
Why do we need that? It should work without preview language. Could you please try rolling these changes back?

Reply:
While I was working on it, somehow it came as VS Helper, but now as I have removed it, it is not throwing an error.